practical/groovy/annotate_dataset.groovy

```diff
@@ -41,15 +46,6 @@ import omero.gateway.model.DatasetData
 import omero.gateway.model.TagAnnotationData
 import omero.log.SimpleLogger
 
-// Setup
-// =====
-
-// OMERO Server details
-HOST = "workshop.openmicroscopy.org"
-PORT = 4064
-// parameters to edit
-USERNAME = "username"
-PASSWORD = "password"
```

Comment:
This script works, but the naming "new tag 2" is a bit odd - why "2" ? https://github.com/ome/training-scripts/pull/78/files#diff-7a868ccfcc6af9ebe2a4d9d083d37c7fR76